discovery/utils.go

```diff
@@ -146,3 +146,19 @@ func SignAnnouncement(signer lnwallet.MessageSigner, pubKey *btcec.PublicKey,
 
 	return signer.SignMessage(pubKey, data)
 }
+
+// remotePubFromChanInfo returns the public key of the remote peer given a
+// ChannelEdgeInfo that describe a channel we have with them.
+func remotePubFromChanInfo(chanInfo *channeldb.ChannelEdgeInfo,
+	msgFlags lnwire.ChanUpdateChanFlags) [33]byte {
```

Comment:
nit: argument should be `chanFlags` as `msgFlags` would indicate a different field